accumulator/pollard.go

```diff
@@ -155,7 +164,7 @@ func (p *Pollard) rem2(dels []uint64) error {
 
 	// get all the swaps, then apply them all
 	swaprows := remTrans2(dels, p.numLeaves, ph)
-	wg := new(sync.WaitGroup)
+	//wg := new(sync.WaitGroup)
```

Comment:
i don't see the value in having this dead code laying around

Reply:
Removed